tunneling/python/app.py

```diff
@@ -0,0 +1,33 @@
+import json
+import urllib
+
+import flask
+import requests
+from flask_cors import CORS
+
+app = flask.Flask(__name__)
+CORS(app)
+
+
+@app.route("/bugs", methods=["POST"])
+def bugs():
+    sentry_host = "sentry.io"
+    known_project_ids = ["123456"]
+
+    envelope = flask.request.data.decode("utf-8")
+    piece = envelope.split("\n")[0]
+    header = json.loads(piece)
+    dns = urllib.parse.urlparse(header.get("dsn"))
+
+    if dns.hostname != sentry_host:
+        return "Invalid sentry host", 400
```

Comment:
I suggest returning 200 here and in the project ID validation (i.e. always return the same data/status to avoid leaking what are the valid host/project IDs.

This is just an example, but people might take it as is or not consider that when building from the example.

Reply:
Good thought. Moved the text to some print statement and just return with 200.